packages/jellyfish-api-core/__tests__/category/loan/getInterest.test.ts

```diff
@@ -72,13 +72,13 @@ describe('Loan getInterest', () => {
     await testing.generate(1)
 
     const vaultAddress = await testing.generateAddress()
-    vaultId = await testing.rpc.loan.createVault({
+    vaultId = await testing.rpc.vault.createVault({
```

Comment:
Revert changes in `loan` test

Reply:
Got it, will only change it after full deprecation.